Doc/whatsnew/3.14.rst

```diff
@@ -691,10 +691,36 @@ Deprecated
     (Contributed by Serhiy Storchaka in :gh:`58032`.)
 
 * :mod:`asyncio`:
-  :func:`!asyncio.iscoroutinefunction` is deprecated
-  and will be removed in Python 3.16,
-  use :func:`inspect.iscoroutinefunction` instead.
-  (Contributed by Jiahao Li and Kumar Aditya in :gh:`122875`.)
+
+  * :func:`!asyncio.iscoroutinefunction` is deprecated
+    and will be removed in Python 3.16,
+    use :func:`inspect.iscoroutinefunction` instead.
+    (Contributed by Jiahao Li and Kumar Aditya in :gh:`122875`.)
+
+  * :mod:`asyncio` policy system is deprecated and will be removed in Python 3.16.
+    The following classes and functions are deprecated:
+
+    * :class:`asyncio.AbstractEventLoopPolicy`
+    * :class:`asyncio.DefaultEventLoopPolicy`
+    * :class:`asyncio.WindowsSelectorEventLoopPolicy`
+    * :class:`asyncio.WindowsProactorEventLoopPolicy`
+    * :func:`asyncio.get_event_loop_policy`
+    * :func:`asyncio.set_event_loop_policy`
+    * :func:`asyncio.set_event_loop`
```

Comment:
```suggestion
    * :class:`!asyncio.AbstractEventLoopPolicy`
    * :class:`!asyncio.DefaultEventLoopPolicy`
    * :class:`!asyncio.WindowsSelectorEventLoopPolicy`
    * :class:`!asyncio.WindowsProactorEventLoopPolicy`
    * :func:`!asyncio.get_event_loop_policy`
    * :func:`!asyncio.set_event_loop_policy`
    * :func:`!asyncio.set_event_loop`
```

Reply:
Please remind me what the exclamation mark is used for?

Reply:
Suppress the link. Kumar suppressed the link for "asyncio.iscoroutinefunction", so I assume that we also want to suppress those links. Technically, they will still be valid until we remove them in 3.16 (or later) so it's up to him though (I just wanted to make the changelog consistent).

Note that we can keep the links for both `asyncio.iscoroutinefunction` and those functions/classes since that would help users clicking on them (we should also add `.. deprecated` directives to those classes and functions by the way)

Reply:
IIRC I had to suppress link for `asyncio.iscoroutinefunction` because it wasn't documented anywhere, i don't think we need to suppress links for the documented ones.

Reply:
Oh! ok, let's keep the links for the documented ones. I thought you wanted to remove the link because you didn't want SEO to pick up the link. Forget this comment then (also forget about adding `deprecated` to the policies since you mention that the entire policy module is deprecated, though users who click on the links or end up on it from a search engine would not necessarily see that the entire module is deprecated; up to you).